flink-container/src/test/java/org/apache/flink/container/entrypoint/StandaloneJobClusterEntryPointTest.java

```diff
@@ -53,6 +55,7 @@ public void testJobGraphRetrieval() throws FlinkException {
 
 		assertThat(jobGraph.getName(), is(equalTo(TestJob.class.getCanonicalName() + "-suffix")));
 		assertThat(jobGraph.getMaximumParallelism(), is(parallelism));
+		assertEquals(jobGraph.getJobID(), new JobID(0, 0));
```

Comment:
use `FIXED_JOB_ID` constant from `StandaloneJobClusterEntrypoint` for the expected value.